lib/main.dart

```diff
@@ -20,23 +20,20 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Flutter Demo',
       theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // TRY THIS: Try running your application with "flutter run". You'll see
-        // the application has a purple toolbar. Then, without quitting the app,
-        // try changing the seedColor in the colorScheme below to Colors.green
-        // and then invoke "hot reload" (save your changes or press the "hot
-        // reload" button in a Flutter-supported IDE, or press "r" if you used
-        // the command line to start the app).
-        //
-        // Notice that the counter didn't reset back to zero; the application
-        // state is not lost during the reload. To reset the state, use hot
-        // restart instead.
-        //
-        // This works for code too, not just values: Most code changes can be
-        // tested with just a hot reload.
         colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
         useMaterial3: true,
+        scaffoldBackgroundColor: const Color(0xFF272727),
```

Comment:
BUG: Die Leiste oben in der Anwendung ist immer Weiß.

1. Diese Leiste existiert nur bei unter menus (Add Tag, Add Transaction, add Goal, Support)
2. Die Leiste soll natürlich auch unseren Scheme nutzen

Reply:
da war ich ja auch noch nicht bei ist also erstmal obsolet